STEERING-COMMITTEE.md

```diff
@@ -162,11 +128,176 @@ There are two ways to raise issues to the steering committee for decision:
+
+### Voting Procedure
+
+Elections will be held using a time-limited
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+[CIVS](http://civs.cs.cornell.edu/) using the Schulze method. The top
+vote-getters will be elected to the open seats, with the exceptions for company
+representation discussed below. This is the same process used for TOC elections.
+
+### Limitations on Company Representation
+
+No more than two seats may be held by employees of the same organization (or
+conglomerate, in the case of companies owning each other). Additionally, the two
+End-User seats may not belong to employees of the same organization. If the
+results of an election result in greater than two employees of the same
+organization, the lowest vote getters from any particular employer will be
+removed until representation on the committee is down to two.
+
+If employers change because of job changes, acquisitions, or other events, in a
+way that would yield more than 2 seats being held by employees of the same
+organization, sufficient members of the committee must resign until only two
+employees of the same employer are left. If it is impossible to find sufficient
+members to resign, all employees of that organization will be removed and new
+special elections held. In the event of a question of company membership (for
+example evaluating independence of corporate subsidiaries) a majority of all
+non-involved Steering Committee members will decide.
+
+## Vacancies
+
+In the event of a resignation or other loss of an elected SC member, the
+candidate with the next most votes from the previous election will be offered
+the seat, provided that person otherwise qualifies to join the SC. This process
+will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible, unless the regular SC election is less than 7 weeks
+away. Eligible voters from the most recent election will vote in the special
+election. Eligibility will not be redetermined at the time of the special
+election. Any replacement SC member will serve out the remainder of the term for
+the person they are replacing, regardless of the length of that remainder.
+
+## Transition
+
+In order to transition from the original, organization-appointed Steering
+Committee structure (the "bootstrap" committee), elections will be phased in
+over 14 months.
+
+In October 2020, four positions will be elected, including three Contributing
+seats and one End-User seat. One Google, one IBM/RedHat, and one VMware seat
+will be retained from the current Steering Committee.
+
+In October or November 2021, the three remaining "bootstrap SC" seats will be
+replaced by elected seats (two Contributing and one End-User).
+
+## Changes to the charter
+
+Changes to the KSC charter may be proposed via a PR on the charter itself.
+Amendments are accepted with majority consent of the committee.
```

Comment:
s/committee./committee as per the Decision Process outlined above.

Reply:
Done.